packages/app/src/components/swap/swap.tsx

```diff
@@ -50,7 +50,7 @@ const Swap: FC<Props> = ({
   const tenderLogo = require("../../images/" + stakers[location.pathname].bwTenderLogo);
 
   const [showConfirm, setShowConfirm] = useState(false);
-  const [isSendingToken, setIsSendingToken] = useState(true);
+  const [isSendingToken, setIsSendingToken] = useState(false);
```

Comment:
Perhaps let's add a comment here which state indicates which token other than that looks good to merge, so feel free to merge after adding the comment !